frontend/sw360-portlet/src/main/webapp/html/components/view.jsp

```diff
@@ -182,13 +180,25 @@
     </table>
 </div>
 
+<span class="pull-right">
+        <select class="toplabelledInput, formatSelect" id="extendedExcelExport" name="extendedExcelExport">
```

Comment:
I do not think that html classes should be komma-seperated, i.e. write `class="toplabelledInput formatSelect"`